CHANGELOG.md

```diff
@@ -3,6 +3,82 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.10
+
+_Dec 1, 2022_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve Ukrainian (uk-UA) and add Urdu (ur-PK) locales
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v6.0.0-alpha.10` / `@mui/x-data-grid-pro@v6.0.0-alpha.10` / `@mui/x-data-grid-premium@v6.0.0-alpha.10`
+
+### Breaking changes
+
+- [DataGrid] Removes `GridCell` fallback to `valueToRender` on `null` children (#7023) @MBilalShafi
+
+  Returning `null` in `column.renderCell` or `column.renderEditCell` now renders an empty cell instead of the default formatted value.
+
```

Comment:
Breaking changes corresponding to https://github.com/mui/mui-x/pull/6985
```suggestion
- The `GridFilterItem['columnField']` was renamed to `GridFilterItem['field']`
- The `GridFilterItem['operatorValue']` was renamed to `GridFilterItem['operator']`
- The `GridFilterItem['operator']` is now required.

```

Reply:
By the way, when a PR introduce a breaking change, a nice trick is to add in the main description of the PR a section 

```
### Changelog
```

All the markdown bellow that will be automatically added to the generated changelog

Reply:
Yeah, just missed it in this PR, thanks for the reminder